roles/init_dbserver/tasks/verify_pgd_settings.yml

```diff
@@ -0,0 +1,48 @@
+---
+- name: Check 'pgd_commit_scopes' dictionary length
+  ansible.builtin.assert:
+    that:
+      - pgd_commit_scopes | length > 0
+    msg: "The 'pgd_commit_scopes' list should have at least one dictionary."
```

Comment:
I don't understand why we would require having at least one commit scope. I think that these tasks `verify_pgd_settings.yml` should only be run if `pgd_commit_scopes | length > 0`